### PR TITLE
deb: add python-rgw packages

### PR DESCRIPTION
**Pull request by liewegas**

### CODE REVIEW

**Comment by liewegas** [comment]
Need to make sure the gitbuilder is happy before merging

**Comment by liewegas** [comment]
or shaman :)

**Comment by liewegas** [comment]
gitbuilders are happy

**Comment by liewegas** [comment]
retest this please

File: ceph.spec.in [hunk 1]
```diff
@@ -517,6 +518,26 @@ Requires:	python%{python3_pkgversion}-rados = %{epoch}:%{version}-%{release}
...
+%package -n python-rgw
+Summary:	Python 2 libraries for the RADOS block device
+Group:		System Environment/Libraries
+License:	LGPL-2.0
+Requires:	librgw1 = %{epoch}:%{version}-%{release}
+Requires:	python-rados = %{epoch}:%{version}-%{release}
+Obsoletes:	python-ceph < %{epoch}:%{version}-%{release}
+%description -n python-rgw
+This package contains Python 2 libraries for interacting with Cephs RADOS
+block device.
```
Comment (cbodley) [inline ceph.spec.in]: copy pasta, has references to rbd

**Pull request merged**